### ISSUE TITLE
Incorrect percentages reported

### ISSUE THREAD

**Issue by donutbrew** [LANL-Bioinformatics/EDGE#19]
In the dev version,  outputMunger_w_temp.pl (line 520) the regex returns the number of matches (1), rather than the number of reads, which leads to incorrect percentages in the top5 table. 

```$NUM_READS_FOR_DOWNSTREAM = ($vars->{AFTERREADS} =~ m/(\d+)/);```
could be
```
$vars->{AFTERREADS} =~ m/(\d+)/); 
$NUM_READS_FOR_DOWNSTREAM = $1;
```
or something like that.

This construction is other places too.

**Comment by kwdavenport** [comment]
Thanks.  We saw this just recently, too. We’ll investigate and make corrections.

Karen
-

**Comment by chienchi** [comment]
bug fixed commit #77a4830
